src/app/pages/dashboard/components/widget-sys-info/widget-sys-info.component.html

```diff
@@ -184,18 +184,20 @@ <h3 fxHide.xs class="mat-card-title-text">
               </div>
 
               <div
-                *ngIf="!data || (isHa && isPassive && haStatus && haStatus !== HaStatusText.HaEnabled)"
+                *ngIf="!data || (isHa && isPassive && !hasHa)"
```

Comment:
`isHa` and `hasHa` are a bit conflicting.
isHa seems to be more like `isHaLicensed`